### PR TITLE
ci: add workflow to redirect people to transifex when they only change translations

### PR DESCRIPTION
**Pull request by rissson**
Small message for contributors to redirect them to transifex when they update translations in this repository.

### CODE REVIEW

File: .github/workflows/translation-advice.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          edit-mode: replace
+          body: |
+            authentik translations instructions
+
+            Thanks for your pull request!
+
+            Authentik translations are handled using
```
Comment (BeryJu) [inline .github/workflows/translation-advice.yml:34]: Very small nitpick; authentik should be stylised lowercase here

File: .github/workflows/translation-advice.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    steps:
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: "github-actions[bot]"
+          body-includes: authentik translations instructions
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
```
Comment (BeryJu) [inline .github/workflows/translation-advice.yml:24]: We could use the bot accounts PAT to create the comment for consistency

File: .github/workflows/translation-advice.yml (hunk 1 above)
Comment (rissson) [reply .github/workflows/translation-advice.yml]: Fixed

File: .github/workflows/translation-advice.yml (hunk 2 above)
Comment (rissson) [reply .github/workflows/translation-advice.yml:24]: Those credentials won't be available on community PRs, will they?

Comment (BeryJu) [reply .github/workflows/translation-advice.yml:24]: oh true I keep forgetting that

**Review by BeryJu — APPROVED** [review]

**Pull request merged**